recipes/tracy/all/conanfile.py

```diff
@@ -0,0 +1,97 @@
+from conans import ConanFile, CMake, tools
+import os
+import textwrap
+
+required_conan_version = ">=1.43.0"
+
+
+class TracyConan(ConanFile):
+    name = "tracy"
+    description = "C++ frame profiler"
+    topics = ("tracy", "profiler", "performance", "gamedev")
+    homepage = "https://github.com/wolfpld/tracy"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = ["BSD 3-clause"]
+    settings = "os", "compiler", "build_type", "arch"
+
+    # Existing CMake tracy options with default value
+    tracy_options = {
+        "enable": ([True, False], True),
+        "callstack": ([True, False], False),
+        "only_localhost": ([True, False], False),
+        "no_broadcast": ([True, False], False),
+        "no_code_transfer": ([True, False], False),
+        "no_context_switch": ([True, False], False),
+        "no_exit": ([True, False], False),
+        "no_frame_image": ([True, False], False),
+        "no_sampling": ([True, False], False),
+        "no_verify": ([True, False], False),
+        "no_vsync_capture": ([True, False], False),
+    }
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        **{k: v[0] for k, v in tracy_options.items()},
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        **{k: v[1] for k, v in tracy_options.items()},
+    }
+
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake"
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def validate(self):
+        if self.settings.compiler.get_safe("cppstd"):
+            tools.check_min_cppstd(self, 11)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+        self.license = "tracy-{}".format(self.version)
```

Comment:
Why is the license being overridden here?
The license of tracy seems to be BSD 3-clause.
I'm not sure why. sorry.

Reply:
That was an erroneous copy & paste, fixed it, thanks.